kDrive/UI/Controller/MainTabViewController.swift

```diff
@@ -243,19 +250,66 @@ extension MainTabViewController: MainTabBarDelegate {
         floatingPanelViewController.trackAndObserve(scrollView: plusButtonFloatingPanel.tableView)
         present(floatingPanelViewController, animated: true)
     }
+
+    func avatarLongTouch() {
+        let viewControllers = viewControllers
```

Comment:
Why do we need to create a second variable `viewControllers`?

Reply:
I do not know, there is no good reason for it. It is probably something that I did not clean up after the PoC stage.